go/oasis-node/cmd/consensus/consensus.go

```diff
@@ -78,24 +81,98 @@ func doSubmitTx(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	if err = client.SubmitTx(context.Background(), &tx); err != nil {
+	return &tx
+}
+
+func doSubmitTx(cmd *cobra.Command, args []string) {
+	if err := cmdCommon.Init(); err != nil {
+		cmdCommon.EarlyLogAndExit(err)
+	}
+
+	conn, client := doConnect(cmd)
+	defer conn.Close()
+
+	tx := loadTx()
+
+	if err := client.SubmitTx(context.Background(), tx); err != nil {
 		logger.Error("failed to submit transaction",
 			"err", err,
 		)
 		os.Exit(1)
 	}
 }
 
+func doShowTx(cmd *cobra.Command, args []string) {
+	if err := cmdCommon.Init(); err != nil {
+		cmdCommon.EarlyLogAndExit(err)
+	}
+
+	sigTx := loadTx()
+
+	var tx transaction.Transaction
+	if err := cbor.Unmarshal(sigTx.Blob, &tx); err != nil {
+		logger.Error("failed to unmarshal unsigned transaction",
+			"err", err,
+		)
+		os.Exit(1)
+	}
+
+	// Pretty print the transaction.
+	fmt.Printf("Signer: %s\n", sigTx.Signature.PublicKey)
+	fmt.Printf("        (signature: %s)\n", sigTx.Signature.Signature)
```

Comment:
SignedTransaction, unlike other SignedXxx structs, doesn't have a PrettyPrint method, and instead there's special treatment in doShowTx.

Reply:
That's a good idea.